src/types.ts

```diff
@@ -1,9 +1,10 @@
 import { Theme } from '@guardian/types';
 
 export type AdTargetingType = {
-    adUnit: string;
+    adUnit?: string;
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    customParams: { [key: string]: any };
+    customParams?: { [key: string]: any };
```

Comment:
`Record<string, unknown>` ?

Reply:
This was the existing type but I agree a Record with an unknown type would be better